modules/blocks/verify.js

```diff
@@ -563,24 +558,24 @@ Verify.prototype.addBlockProperties = function(block) {
  * @returns {Object} Block object reduced
  */
 Verify.prototype.deleteBlockProperties = function(block) {
-	const reducedBlock = JSON.parse(JSON.stringify(block));
+	const reducedBlock = Object.assign({}, block);
```

Comment:
its completely the wrong way to clone a object, if you do stringify and parse then it would change the object properties like, if its a bignum then it apply string and convert, so the intension is to clone not to modify the state, so I have changed it to the correct way